pkg/apply/destroyer_test.go

```diff
@@ -43,13 +45,10 @@ func TestDestroyerCancel(t *testing.T) {
 			expectRunTimeout: true,
 			runTimeout:       2 * time.Second,
 			testTimeout:      30 * time.Second,
-			invInfo: inventoryInfo{
-				name:      "abc-123",
-				namespace: "test",
-				id:        "test",
-				set: object.ObjMetadataSet{
-					testutil.ToIdentifier(t, resources["deployment"]),
-				},
+			invObjs: object.ObjMetadataSet{
```

Comment:
This minor refactor could be done in a separate PR. It would just modify this test to build the inventoryInfo in the test body, instead of here in the testCases.